src/card.css

```diff
@@ -0,0 +1,99 @@
+.card {
+  width: 200px;
+  background-color: var(--color-white);
+  border: 1px solid var(--color-concrete);
+}
+  
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin-left: calc(50% - 160px / 2);
+  margin-top: 33px;
+  margin-bottom: 36px;
+}
+  
+.card__stars_box {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  margin-right: calc(50% - 166px / 2);
+  margin-left: calc(50% - 166px / 2);
+}
+  
+.card__title {
+  font-size: 12px;
+  font-weight: 500;
+  color: #060b35;
+  height: 36px;
+  line-height: 18px;
+  margin: 0;
```

Comment:
Why did you add margin: 0 here? if you need to rest some margins for some elements, use reset.css file